python_src/src/lamp_py/migrations/versions/performance_manager_prod/001_5d9a7ee21ae5_initial_prod_schema.py

```diff
@@ -371,9 +371,7 @@ def upgrade() -> None:
         sa.UniqueConstraint(
             "service_date",
             "route_id",
-            "direction_id",
-            "start_time",
-            "vehicle_id",
+            "trip_id",
```

Comment:
We have examples in our test set where the same `trip_id` continues across more than one Green Line `route_id` for the same `start_date`. 

I'm not sure how common this is, but it does occur, and I believe the unique trip designations we are looking for should include `route_id` as a guarantee.

Reply:
Comment has been added to the function call to describe this behavior

Reply:
If you have details about when you've seen that, it would be helpful: I don't think that should be happening (at least not within a single start_date).

Reply:
Out flat file test data is a random sample from May 8th of this year. Two trip ids exhibit this behavior (ADDED-1581518542, ADDED-1581518549). 

CSV file with the GTFS-RT Vehicle Position data is attached. 

[non_unique_trip_id.csv](https://github.com/mbta/lamp/files/12432335/non_unique_trip_id.csv)

Reply:
Ah, something like this makes sense for ADDED trips (although probably that reflects a bug in RTR). I'm still not sure why `route_id` needs to be included in the unique constraint though (given that it's possible for some of the other items in there to be non-unique as well), but I'll defer to the Lamp team.